types/index.d.ts

```diff
@@ -1055,10 +1049,10 @@ declare namespace math {
      * if false (default) the principal root is returned.
      * @returns Returns the cubic root of x
      */
-    cbrt(x: number, allRoots?: boolean): number
-    cbrt(x: BigNumber): BigNumber
-    cbrt(x: Complex, allRoots?: boolean): Complex
-    cbrt(x: Unit): Unit
+    cbrt<T extends number | BigNumber | Complex | Unit>(
```

Comment:
The optional argument `allRoots` is only applicable for complex numbers, can you keep that as a separate definition?